index.html

```diff
@@ -10,21 +10,30 @@
 </head>
 
 <body>
-    <header class="white">
-        <a class="logo" href="#">My logo</a>
-        <nav class="nav-bar">
-            <div class="bars"><img src="images/menu.png" alt="menu navbar" /></div>
-            <div class="desk-menu">
-                <ul>
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul></div>
-        </nav>
+    <header class="menu-animation">
+        <div class="container">
+            <div class="logoforJS">
+               <h2>My logo</h2>
+            </div> 
+
+            <div class="nav-toggle" id="navToggle">
+                <img src="./images/menu.png" alt="hamburger-menu"/>
+            </div>        
+           
+            <nav class="desktop-nav">            
+                <ul class="links">
+                    <li class="close">X</li>                    
+                    <li class="nav-items"><a href="#porto">Portfolio</a></li>
+                    <li class="nav-items"><a href="#aboutMe">About</a></li>
+                    <li class="nav-items"><a href="#contact-me">Contact</a></li>
+                </ul>

```

Comment:
- [ ] your mobile menu popup works well. please like the previous reviewer noted, kindly fix the missing desktop navigation links `(portfolio, about me, contact)` to ease the accessibility of the application.

Reply:
hello, if you click on the links, it takes you straight to the part of the page that consista of that information, Both on the desktop and mobile version. May you kindly double check, 

- In the previous review, the issue seemed to be that my nav was missing. I then included it after the request of the change. I found out the list waas missing becauseof  what i placed in my media query. May you please check how the link looks now compared to before. 
- Also let me know if i am not understanding your comments properly.

Reply:
hello, if you click on the links, it takes you straight to the part of the page that consista of that information, Both on the desktop and mobile version. May you kindly double check, 

- In the previous review, the issue seemed to be that my nav was missing. I then included it after the request of the change. I found out the list waas missing becauseof  what i placed in my media query. May you please check how the link looks now compared to before. 
- Also let me know if i am not understanding your comments properly.

Reply:
check if you pushed the code correctly. i cant see any file changes so I thought you forgot to make the changes. please verify it

Reply:
thanks ive made another request, thanks.